src/core/send.c

```diff
@@ -1283,7 +1283,7 @@ QuicSendFlush(
         }
 
 #if DEBUG
-        CXPLAT_DBG_ASSERT(++DeadlockDetection < 100);
+        CXPLAT_DBG_ASSERT(++DeadlockDetection < 1000);
```

Comment:
Should this be `QUIC_MAX_FRAMES_PER_PACKET * QUIC_MAX_DATAGRAMS_PER_SEND * 2` rather than a magic constant?

Reply:
No. .NET folks were seeing this hit in their stress tests and I just want to fix that. I don't expect to the other things to change really any more, so I'm fine with it this way.